src/components/Header.tsx

```diff
@@ -1,36 +1,37 @@
-import logo from '../assets/logo/logo-dark.svg';
-import { useTranslation } from 'react-i18next';
-import MultiLanguageSelect from './MultilanguageSelect';
-import HamburgerSelect from './HamburgerSelect';
+import logo from "../assets/logo/logo-dark.svg";
+import { useTranslation } from "react-i18next";
+import MultiLanguageSelect from "./MultilanguageSelect";
+import HamburgerSelect from "./HamburgerSelect";
```

Comment:
please make sure you have the same settings, so we don't have these changes in every pr